sdks/python/apache_beam/testing/load_tests/build.gradle

```diff
@@ -29,18 +29,20 @@ def loadTestArgsProperty = "loadTest.args"
 
 def runnerProperty = "runner"
 
-task run(type: Exec) {
-    dependsOn 'setupVirtualenv'
-    dependsOn 'installGcpTest'
-
+task run(type: Exec, dependsOn: installGcpTest) {
     def loadTestArgs = project.findProperty(loadTestArgsProperty) ?: ""
 
     def runnerArg = project.findProperty(runnerProperty) ?: ""
     if (runnerArg == 'DataflowRunner' || runnerArg == 'TestDataflowRunner') {
-        dependsOn 'sdist'
-        loadTestArgs +=" --sdk_location=${project.buildDir}/apache-beam.tar.gz"
+        def sdistPath = "${project.rootDir}/sdks/python/apache_beam/testing/load_tests/build/apache-beam.tar.gz"
+        println sdistPath
```

Comment:
Are these `println`s debugging leftovers? Did you intend to leave them?

Reply:
Thank you. All of those prints should be deleted, I forgot about them.